Add R Lille

mcanouil opened the pull request:
Add R Lille information: website, meetup and twitter
Add April Meetups.

csgillespie commented:
@mcanouil Thanks for this. I've tweaked your PR.

1. Just give a single link to the group (remove meetup.com).
2. Delete the events

You could add
  
## 2021 `r get_btn()` {-}

### April

  * April 15: [R Lille Meetup](https://www.meetup.com/R-Lille/events/277031496/). Lille, France. [\@rlille_rug](https://twitter.com/rlille_rug).
  * April 29: [R Lille Meetup](https://www.meetup.com/R-Lille/events/277132394/). Lille, France. [\@rlille_rug](https://twitter.com/rlille_rug).

to the virtual meet-ups?

mcanouil commented:
My bad, I did not notice the virtual events file.
I added the two meetups in the proper file and changed the main URL to target a more international website, i.e., Meetup page.

csgillespie commented:
Looks good! Thanks
